src/metabase/api/timeline_event.clj

```diff
@@ -0,0 +1,79 @@
+(ns metabase.api.timeline-event
+  "/api/timeline-event endpoints."
+  (:require [compojure.core :refer [DELETE GET POST PUT]]
+            [metabase.api.common :as api]
+            [metabase.models.collection :as collection]
+            [metabase.models.timeline :refer [Timeline]]
+            [metabase.models.timeline-event :refer [TimelineEvent]]
+            [metabase.util :as u]
+            [metabase.util.date-2 :as u.date]
+            [metabase.util.i18n :refer [tru]]
+            [metabase.util.schema :as su]
+            [schema.core :as s]
+            [toucan.db :as db]))
+
+(api/defendpoint POST "/"
+  "Create a new [[TimelineEvent]]."
+  [:as {{:keys [name description timestamp time_matters timezone icon timeline_id archived] :as body} :body}]
+  {name         su/NonBlankString
+   description  (s/maybe s/Str)
+   ;; todo: find schema for timestamps?
```

Comment:
I didn't see one in the code base, I could have missed it or you might need to make one.

Reply:
I ended up making one. I guess we forgot to put it here. Good call

Reply:
Cool, sounds like hooking up the timestamp schema is all that's needed now. I'll appove